src/odemis/gui/util/img.py

```diff
@@ -1031,8 +1027,8 @@ def draw_scale(ctx, value_range, client_size, orientation, tick_spacing,
 
             if prev_lpos >= lpos + 20 or i == 0 or i == len(tick_list):
                 if mirror:
-                    ctx.move_to(scale_width - lbl_width - 9, client_size[1] - lpos)
-                    ctx.show_text(label)
+#                     ctx.move_to(scale_width - lbl_width - 9, client_size[1] - lpos)
+#                     ctx.show_text(label)
```

Comment:
why did you leave this here?

Reply:
Because currently the function shows the text or not based on whether the scale is shown at the top or bottom (parameter `mirror`). I don't like such arbitrary behaviour. So I have in mind that one day there would be a separate parameter to show or not the text. In such case, this code will be used.